controller/testdata/live-deployment-env-vars.yaml

```diff
@@ -137,7 +137,7 @@ spec:
         - env:
             - name: SOME_ENV_VAR
               value: some_value
-          image: 'gcr.io/heptio-images/ks-guestbook-demo:0.1'
+          image: 'quay.io/argoprojlabs/argocd-e2e-container:0.1'
```

Comment:
Here the `argocd-e2e-container` image is used with a 0.1 tag

Reply:
these are by default.